tests/serverpod_test_server/test_e2e/auth_module_test.dart

```diff
@@ -172,4 +172,66 @@ void main() {
       expect(result, equals(true));
     });
   });
+
+  group('Given signed in user that wants to alter its user info', () {
+    setUp(() async {
+      var response = await client.authentication.authenticate(
+        'test@foo.bar',
+        'password',
+      );
+      assert(response.success, 'Failed to authenticate user');
+      await client.authenticationKeyManager
+          ?.put('${response.keyId}:${response.key}');
+      assert(
+          await client.modules.auth.status.isSignedIn(), 'Failed to sign in');
+    });
+
+    tearDown(() async {
+      await client.authenticationKeyManager?.remove();
+      await client.authentication.removeAllUsers();
+      await client.authentication.signOut();
+      assert(
+        await client.modules.auth.status.isSignedIn() == false,
+        'Still signed in after teardown',
+      );
+    });
+    test('when changing its user name then user name is updated.', () async {
+      final currentUserInfo = await client.modules.auth.status.getUserInfo();
+      expect(currentUserInfo, isNotNull,
+          reason: 'We expect to get UserInfo back from the auth module');
+      final String updatedFullName = '${currentUserInfo?.fullName}updated';
+      expect(currentUserInfo?.userName, isNot(equals(updatedFullName)),
+          reason: 'The new name should not be the same as the current for the '
+              'test to acutally test something.');
```

Comment:
Suggestion: Remove this expect.

This expect is validating that the dart "set" function is working correctly. We don't need to validate that in this test.

Reply:
Same for other test.

Reply:
Makes sense, will fix.